mods/xamionex.UnholyTrinity/mod/scripts/vscripts/lobby/_lobby.gnut

```diff
@@ -19,6 +19,7 @@ void function Lobby_Init()
 	SetPlaylistVarOverride( "scorelimit", "200" )
 	SetPlaylistVarOverride( "timelimit", "30" )
 	ServerCommand("slide_step_velocity_reduction -45")
+	ServerCommand("sv_gravity 800")
```

Comment:
Doesn't this need sv_cheats 1 tho?

Reply:
Idea: Make it set sv_cheats 1, do the gravity thingy, and then set it to 0? I've seen it done before and I noticed a difference when testing it...

Reply:
I'll change the code in a bit, if needed I have an idea on how to do this and add 50 extra gravity xD.

Reply:
you could just do this
```suggestion
ServerCommand("sv_cheats 1")
ServerCommand("sv_gravity 800")
ServerCommand("sv_cheats 0")
```

Reply:
Yeah but that's not really safe as in when the server starts *someone* will know that sv_cheats enables for a split second and try something

Reply:
set player gravity on spawn or just change the convar instead of running commands? `SetConVarInt("sv_gravity", "800")` should work?

Reply:
well a split second of sv_cheats wouldn't exist since it all should happen on one frame.
the set convar idea is better.